source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -242,11 +243,25 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         return content;
     }
 
-    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url)
+    winrt::Image CreateBackgroundImage(winrt::AdaptiveRenderContext const& renderContext, winrt::hstring const& url, winrt::TileControl const& tileControl)
     {
         try
         {
             auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+            auto imagePath = HStringToUTF8(imageUrl.Path());
+            bool isImageSvg = imagePath.find("svg") != std::string::npos;
+
+            if (isImageSvg)
+            {
```

Comment:
Update the check for `isImageSvg` as some png urls could also have "svg" somewhere buried in the url.

![image](https://github.com/microsoft/AdaptiveCards/assets/165726045/1068aa84-6dfe-4f85-bbdb-b70fd1fa6f94)

https://microsoft-my.sharepoint-df.com/:u:/p/cmcguire/EaW8w9ab3UNLq3q0C2JUE6sBFIpy8MIE4ONpfJEZO2LslA?e=60dkYj